FrontPanel/FrontPanel.cpp

```diff
@@ -177,6 +178,9 @@ namespace WPEFramework
             registerMethod(METHOD_FP_IS_24_HOUR_CLOCK, &FrontPanel::is24HourClockWrapper, this);
             registerMethod(METHOD_FP_SET_CLOCKTESTPATTERN, &FrontPanel::setClockTestPatternWrapper, this);
 
+            Core::JSONRPC::Handler& version2 = JSONRPC::CreateHandler({ 2 }, *this);
```

Comment:
Pls use the abstractPlugin registerMethod() and pass in the supported versions - https://github.com/rdkcentral/rdkservices/commit/a15b9db120d9ac710b05aa76ccf3b12f983b3fa2